frontend/src/css/modules/variables.scss

```
@@ -61,6 +61,10 @@ $loader-dot-color: #ffaf4b;
 
 $border-color-light: #e3e3e3;
 
+/*announcement banner*/
```

Stylelint (17.12.0)

[error] Expected whitespace after "/*" (comment-whitespace-inside)

(comment-whitespace-inside)